diesel_compile_tests/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-04-01
+nightly-2020-03-19
```

Comment:
Why do we go back from `nightly-2020-04-01` to `nightly-2020-03-19`?

Reply:
I was trying to rollback to a version that does not have the following issue https://github.com/rust-lang/rust/issues/70814
but this version seems to still have the issues so it is probably easier to allow unused_braces.

Reply:
This was also the last build that had clippy and rustfmt before 04-01, I picked the most recent one that had both and did a work around for the nightly regression